packages/test/src/test-integration-workflows.ts

```diff
@@ -282,3 +283,34 @@ test('Activity initialInterval is not getting rounded', async (t) => {
   const retryPolicy = activityTaskScheduledEvents?.activityTaskScheduledEventAttributes?.retryPolicy;
   t.is(tsToMs(retryPolicy?.initialInterval), 50);
 });
+
+test('Start of workflow is delayed', async (t) => {
+  const { startWorkflow } = helpers(t);
+  // This workflow never runs
+  const handle = await startWorkflow(runTestActivity, {
+    startDelay: '5678s',
+  });
+  const { events } = await handle.fetchHistory();
+  const workflowExecutionStartedEvent = events?.find((ev) => ev.workflowExecutionStartedEventAttributes);
+  console.log(workflowExecutionStartedEvent);
+  const startDelay = workflowExecutionStartedEvent?.workflowExecutionStartedEventAttributes?.firstWorkflowTaskBackoff;
```

Comment:
nit: you could use `!` instead of `?` here and below since the assertion on the next line clearly expects it not to be `undefined`.

Reply:
I'm not sure that is true, `tsToMs` already takes care of the case when is `undefined`, and if the test failed, it will be `undefined`, and we only know that at runtime...